### ISSUE TITLE
language independent field definitions for dexterity

### ISSUE THREAD

**Issue by agitator** [plone/plone.app.event#170]

**Comment by agitator** [comment]
context aware adapters work, but still fighting timezone problems with post_processing

**Comment by datakurre** [comment]
@agitator Nice to hear. @thet might be interested in discussing about post_processing; If I remember correctly, the post processing was almost done in the behavior adapter at one point, but it was later refactored into z3c.form event handler. One main issue has been that post_processing can only truly happen after all field values are known.

**Comment by agitator** [comment]
talked to him at the aplinecitysprint, dug further into it, seems to be connected with the special post processing of dates within plone.app.event

**Comment by datakurre** [comment]
@agigator

Yes, there's event handler, which is only called after successful
z3c.form POST submission with the saved values. It's far from perfect
solution, but the problem has been hard and we haven't figured out a
better yet :(

Probably it should somehow be refactored back to behavior adapter, so
that it would be z3c.form independent.


agitator wrote

**Comment by jensens** [comment]
Closing as outdated. Language independent fields. p.a.multilingual handles this differently in Plone 6.
If you think this issue is still relevant please reopen it. Thank you for the original report!
